cirq/google/serializable_gate_set.py

```diff
@@ -127,7 +127,8 @@ def serialize_op(self,
                     proto_msg = serializer.to_proto(gate_op, msg)
                     if proto_msg is not None:
                         return proto_msg
-        raise ValueError('Cannot serialize op of type {}'.format(gate_type))
+        raise ValueError('Cannot serialize op {!r} of type {}'.format(
```

Comment:
Nit: Generally you want to put reprs at the end of the error message, because they can be long.